packages/web-components/src/components/ic-button/ic-button.stories.mdx

```diff
@@ -1321,3 +1321,43 @@ export const defaultArgs = {
     `}
   </Story>
 </Canvas>
+
+### File Upload
+
+<Canvas>
+  <Story name="File Upload" parameters={{ loki: { skip: true } }}>
+    {html`
+      <script>
+        const fileBtn = document.querySelector("#file-button");
+        fileBtn.addEventListener("icFileSelection", (ev) => {
+          console.log(ev.detail);
+        });
+      </script>
+      <div style="padding: 6px">
+        <ic-button
+          variant="primary"
+          id="file-button"
+          file-upload="true"
+          accept=".doc, text/plain, .json"
+          >Upload File true</ic-button
+        >
+      </div>
+      <div style="padding: 6px">
```

Comment:
I'm not sure this storybook needs the 'false' and 'no prop' examples - since file-upload is an optional property, the other stories show us how buttons without file-upload behave